testing/regress/hpcc/regression/suite.py

```diff
@@ -126,8 +126,17 @@ def addFileToSuite(self, eclfile):
             versions = eclfile.getVersions()
             versionId = 1
             for version in versions:
-                files.append({'basename':basename, 'version':version,  'id':versionId })
-                versionId += 1
+                if 'no'+self.name in version:
+                    # Exclude it from this target
+                    pass
+                else:
+                    # Remove exclusion key(s) from version string
+                    config = getConfig();
```

Comment:
What does the list of clusters mean?  Is it the platform type (e.g., hthor/thor/roxie) or is it the names of the clusters - e.g., could be thor1 mythor etc?